README.md

```diff
@@ -117,6 +124,8 @@ require('core-js/build')({
   // ...
 });
 ```
+If your webpack build includes something like UglifyJs, then the excluded parts will be entirely stripped from the resulting code using dead code elimination.
```

Comment:
Ahyes, i see why that would be confusing. We dont want webpack removing our input code :-) Will update.